jsonthebuilder/src/App.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import './App.css';
+import ResumeContainer from './components/ResumeContainer';
```

Comment:
Prefer adding some comments to split the different app imports.
It will be easier to distinguish them and keep the code maintanable.

```suggestion
// Styles
import './App.css';

// Components
import ResumeContainer from './components/ResumeContainer';
```